src/erlcloud_ddb2.erl

```diff
@@ -862,13 +870,15 @@ undynamize_stream_view_type(<<"NEW_IMAGE">>, _) -> new_image;
 undynamize_stream_view_type(<<"OLD_IMAGE">>, _) -> old_image;
 undynamize_stream_view_type(<<"NEW_AND_OLD_IMAGES">>, _) -> new_and_old_images.
 
--spec undynamize_stream_specification(jsx:json_term(), undynamize_opts()) -> stream_specification().
+-spec undynamize_stream_specification(jsx:json_term(), undynamize_opts()) -> stream_specification_out().
 undynamize_stream_specification(Json, Opts) ->
     case proplists:get_value(<<"StreamEnabled">>, Json, false) of
         false ->
-            false;
```

Comment:
Ah, this one is actually for backward compatibility, as the initial implementation of the interface was naive, see https://github.com/erlcloud/erlcloud/pull/195 ([here](https://github.com/erlcloud/erlcloud/pull/195/files#diff-7e436c0c4b4861e9228d1dfd9bdf109eR54)). It looks like `#ddb2_stream_description` was introduced in https://github.com/erlcloud/erlcloud/pull/499 for DDB backup APIs ([here](https://github.com/erlcloud/erlcloud/commit/b70efdcdbac1f8723ae920b1731d512dd8eb9618#diff-7e436c0c4b4861e9228d1dfd9bdf109eR294-R303)). I think we should probably not touch the core describe_table API, I know for sure it would break our code internally. Sorry for not investigating further before.

Reply:
I'll revert the change to the change, since it seems acceptable.

Reply:
I deleted the commit (it's as if it never existed... poof!).